rollup-examples.config.js

```diff
@@ -1,12 +1,30 @@
 var path = require( 'path' );
 var fs = require( 'fs' );
+var isModuleCanonical = require( './utils/isModuleCanonical.js' );
 
 // Creates a rollup config object for the given file to
 // be converted to umd
 function createOutput( file ) {
 
 	var inputPath = path.resolve( file );
 	var outputPath = inputPath.replace( /[\\\/]examples[\\\/]jsm[\\\/]/, '/examples/js/' );
+	var relativePath = path.relative( './examples/jsm', file ).replace( /^\.[\/\\]/, '' );
+
+	var banner = '';
+	if ( isModuleCanonical( relativePath ) ) {
+
+		banner = [
+			'/**',
+			` * Generated from original source in "examples/js/${ relativePath.replace( /\\/g, '/' ) }".`,
```

Comment:
I think the banner text (here and below in `modularize.js`) is backwards — should say `examples/jsm` here, right?